src/bio/bio_xstream.c

```diff
@@ -512,6 +512,12 @@ is_server_started(void)
 	return nvme_glb.bd_started;
 }
 
+inline d_list_t *
```

Comment:
why inline?

Reply:
inline is supposed to be more efficient for such kind of short function.